packages/fxa-auth-server/lib/senders/emails/locales/ar.ftl

```diff
@@ -0,0 +1,8 @@
+# cadReminderFirst 
```

Comment:
Oooh I hope you didn't manually copy each of these over by hand (or I hope it didn't take too long). 😅

I believe these get automatically converted for us when we build - we have an l10n team that helps with translations. I see we only have the English version committed in `fxa-settings`:
![image](https://user-images.githubusercontent.com/13018240/126570334-676e65ff-f0b7-43e7-873b-588db85374f8.png)

I am not as familiar with Fluent workings as some others may be, but it seems like we only want to commit an FTL file per template/partial? @jodyheavener does that seem right to you? 🤔 I tried to find documentation on Fluent and how everything hooks up, but only found docs for GetText.